crates/ruff_python_formatter/src/comments/visitor.rs

```diff
@@ -1,17 +1,23 @@
-use crate::comments::node_key::NodeRefEqualityKey;
-use crate::comments::placement::place_comment;
-use crate::comments::{CommentLinePosition, CommentsMap, SourceComment};
+use std::iter::Peekable;
```

Comment:
Are you using `rustfmt` nightly? or do you group the use statements manually?

Reply:
I use "Optimize Imports" in IntelliJ. It's arguably a bad habit but it helps a lot with refactors since it automatically cleans up unused imports and others.